homeassistant/components/modbus/validators.py

```diff
@@ -60,6 +60,9 @@
         "slave_count",
         "swap_byte",
         "swap_word",
+        "register_size_2",
+        "register_size_4",
```

Comment:
This information is in the new struct, please do not duplicate the same information.